### ISSUE TITLE
Questions about the tensorflow version

### ISSUE THREAD

**Issue by Z-Zili** [tancik/StegaStamp#57]
Tensorflow is required to be 1.13, but your CUDN version does not support TensorFlow 1.13, in addition to reducing the CUDN version, is there a solution?

**Comment by shiqueenTang** [comment]
just add .compat.v1 like I did

**Comment by Z-Zili** [comment]
Can you run the program when you use “.compat.v1”? I also saw this solution when I first ran the program, but after experimenting with this solution, other problems arose.

**Comment by shiqueenTang** [comment]
yes, after couple of .compat.v1 the program runs perfectly, you just need to have extra patience when dealing with tf 1.x
